Changes

```diff
@@ -2865,7 +2865,7 @@ OCaml 4.13.0 (24 September 2021)
   (Xavier Leroy, report by Richard Bornat, review by Stephen Dolan
    and Mark Shinwell)
 
-- #10371: no longer generatd useless `.cds` file when using
+- #10371: no longer generated useless `.cds` file when using
```

Comment:
I think "generate" was intended.  You need a verb here.  Or, better: "stop generating".

Reply:
Corrected.